config.example.toml

```diff
@@ -1,3 +1,8 @@
+# Include any additional configuration file(s)
+# Note that this config file overrides the includes
+# and includes that are on the left override the ones on the right in this array
```

Comment:
Thanks for your interest in this issue.

This part caught my attention. If consistency with other tools is important, it might be good to look at how other tools handle this. It's my understanding that Git, Vim, Bash, and others don't have the main configuration file take precedence. Instead, settings at the bottom of those configuration files take precedence over settings at the tops of those files, and included files are treated as though their contents are inlined.

For example, with the following setup, the effective email address would be _included@example.com_:

```gitconfig
// .gitconfig
[user]
  email = main@example.com

[include]
  path = .gitconfig.included
```

```gitconfig
// .gitconfig.included
[user]
  email = included@example.com
```

However, if the include is moved up, the effective email address would be _main@example.com_:

```gitconfig
// .gitconfig
[include]
  path = .gitconfig.included

[user]
  email = main@example.com
```

```gitconfig
// .gitconfig.included
[user]
  email = included@example.com
```

As far as I know, the same principle is true for Bash, Vim, and others. Settings at the bottom override settings at the top, and included files are treated as though their contents are right there.

Reply:
This can't be done, not without a breaking change in the way the misc config entries are handled. 
This is because as of now these entries are handled in the global scope, and adding an `[include]` section in between these breaks the following ones, because there is currently no way to specify a return to global scope in TOML files.  

The simplest way to fix this is to add a section named `[misc]` and move all the currently-in-global-scope entries to it.
The problem with this approach is that it breaks all the previously-created config files.

Reply:
@DottoDev since you are the maintainer of this repository, I'm asking you to tell me how to proceed.

If we are going to migrate all currently-in-global-scope entries to the new `[misc]` section, I'm available to adapt `topgrade` to auto migrate the config file.

Reply:
I've just pushed a commit to resolve this. I've already tested it myself, but you might want to check it out too @openjck.